src/sql/base/browser/ui/editableDropdown/dropdown.ts

```diff
@@ -211,6 +211,7 @@ export class Dropdown extends Disposable {
 				this._filter.filterString = e;
 				this._layoutTree();
 			}
+			this._onValueChange.fire(e);
```

Comment:
In `onSelectionChange` above we explicitly set `this.value` to the event's value. Does not doing that here cause there to be an inconsistency between the value shown in the `onValueChange` event vs the value from accessing `this.value`?

Reply:
I did a quick investigation and this will not have impact on the database dropdown. please set this.value before broadcasting the value change event. then you should be good